pcdsdevices/usb_encoder.py

```diff
@@ -0,0 +1,52 @@
+"""
+Class to handle USB encoders
+"""
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal
+
+from .interface import BaseInterface
+
+
+class USBEncoder(BaseInterface, Device):
+    """
+    USB encoder class
```

Comment:
Can we be more specific: which USB encoder? Which model or which IOC support provides these PVs?

Reply:
I ve added some info and a url

Reply:
Great - I think then I'd recommend that this be called something like: `UsDigitalUsbEncoder`
We might not ever get another USB encoder reader, but if we do, it'd be better not have to rename this one.